### PR TITLE
feat(migrations): add 0.16 migration script

### PR DESCRIPTION
**Pull request by ecrupper**
Adding migration script for users adjusting to new `trusted` implementation. Two actions:

1. Set all repos `trusted` field to `false`.
2. Given a slice of privileged images and number of days since last usage, set repos that have used privileged images during that time `trusted` field to `true`.
3. Additional option to limit personal org repos regardless of privileged image usage.

### CODE REVIEW

File: migrations/v0.16/Makefile [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+#
+# Usage: `make build-windows`
+.PHONY: build-windows
+build-windows:
+	@echo
+	@echo "### Building release/windows/amd64/vela-migration binary"
+	GOOS=windows CGO_ENABLED=0 GOARCH=amd64 \
+		go build -a \
+		-o release/windows/amd64/vela-migration \
+		github.com/go-vela/community/migrations/v0.14
```
Comment (plyr4) [inline migrations/v0.16/Makefile:136]:
```suggestion
		github.com/go-vela/community/migrations/v0.16
```

Comment (plyr4) [reply migrations/v0.16/Makefile:136]: search+replace v0.14 in the Makefile?

File: migrations/v0.16/go.mod [hunk 2]
```diff
@@ -0,0 +1,47 @@
+module github.com/go-vela/community/migrations/v0.14
```
Comment (plyr4) [inline migrations/v0.16/go.mod:1]: v0.14?

File: migrations/v0.16/go.mod [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+go 1.17
```
Comment (plyr4) [inline migrations/v0.16/go.mod:3]: go 1.19? not necessary but easy to bump

File: migrations/v0.16/main.go [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+)
+
+func main() {
+	app := cli.NewApp()
+
+	// Utility Information
+
+	app.Name = "vela-migration"
+	app.HelpName = "vela-migration"
+	app.Usage = "Vela utility used to migrate from v0.13.x to v0.14.x"
```
Comment (plyr4) [inline migrations/v0.16/main.go:25]: v0.15.x to v0.16.x

File: migrations/v0.16/main.go [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+	// Utility Flags
+
+	app.Flags = []cli.Flag{
+
+		// Action Flags
+
+		&cli.BoolFlag{
+			EnvVars: []string{"VELA_ACTION_ALL", "ACTION_ALL"},
+			Name:    "action.all",
+			Usage:   "enables running all actions for v0.14.x",
```
Comment (plyr4) [inline migrations/v0.16/main.go:49]: v0.14.x

**Review by plyr4 — CHANGES_REQUESTED** [review]

**Review by plyr4 — DISMISSED** [review]

**Review by wass3r — APPROVED** [review]

**Review by plyr4 — APPROVED** [review]

**Pull request merged**